server/paymentProviders/stripe/creditcard.js

```diff
@@ -147,7 +149,7 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
   if (paymentIntent.status !== 'succeeded') {
     logger.error('Unknown error with Stripe Payment Intent.');
     logger.error(paymentIntent);
-    throw new Error('Unknown error with Stripe. Please contact support.')();
+    throw new Error(UNKNOWN_ERROR_MSG);
```

Comment:
There was something strange on this line `Error(...)()`, not sure if it worked

Reply:
Saw that too this afternoon.